src/client/components/Molecules/Navbar/Navbar.jsx

```diff
@@ -2,27 +2,51 @@ import css from './Navbar.module.css';
 import { NavigationLink } from '#atoms/NavigationLink/NavigationLink.jsx';
 
 const Navbar = () => {
+  const links = [
+    {
+      path: '/categories/:categoryName',
+      title: 'Categories',
+      icon: false,
+    },
+    {
+      path: '/add',
+      title: 'Add recipes',
+      icon: false,
+    },
+    {
+      path: '/my',
+      title: 'My recipes',
+      icon: false,
+    },
+    {
+      path: '/favorite',
+      title: 'Favorite',
+      icon: false,
+    },
+    {
+      path: '/shopping-list',
+      title: 'Shopping list',
+      icon: false,
+    },
+    {
+      path: '/search',
+      title: 'Search',
+      icon: true,
+    },
+  ];
```

Comment:
Move this data out of our Component or even make `.json` file for it 👍

Reply:
done

Reply:
✅